discord/user.py

```diff
@@ -389,6 +390,10 @@ async def edit(self, *, username: str = MISSING, avatar: bytes = MISSING) -> Cli
             payload['username'] = username
 
         if avatar is not MISSING:
+            if "http" in str(avatar):
```

Comment:
```suggestion
            if str(avatar).startswith("http"):
```
This change would make the code a bit less ambiguous while achieving the same effect. 

Though I wonder if there's a better way to instead check for the absence of a bytes-like object being passed rather than a string beginning with "http" being passed.

Reply:
It would definitely be more effect if there was a way to check for the absence of a bytes-like object. I guess you could check the type of the variable being passed?